README.md

```diff
@@ -3,17 +3,28 @@
 
 [![Build Status](https://circleci.com/gh/iodide-project/pyodide.png)](https://circleci.com/gh/iodide-project/pyodide)
 
-The Python scientific stack, compiled to WebAssembly.
+Pyodide is cPython compiled to WebAssembly and running in the browser.  It can run any pure Python package, and 
```

Comment:
> It can run any pure Python package,

This isn't really true.  Anything with networking, such as Requests, would be right out, for example.